packages/ios-client/Sources/FishjamClient/models/SimulcastConfig.swift

```diff
@@ -2,50 +2,34 @@
 /// `"h"` - original encoding
 /// `"m"` - original encoding scaled down by 2
 /// `"l"` - original encoding scaled down by 4
-public enum TrackEncoding: Int, CustomStringConvertible, Codable {
+public enum TrackEncoding: Int, CustomStringConvertible, Codable, CaseIterable {
     case l = 0
     case m
     case h
 
+    static private let descriptionMap: [Self: String] = [.l: "l", .m: "m", .h: "h"]
```

Comment:
The old version had a benefit of compiler warning us of any new cases. If we add new `TrackEncoding` there will be no warning now.

Reply:
There will be no more cases in this enum.